### PR TITLE
PCSensor.FootSwitch version 7.3.4

### PR DESCRIPTION
**Pull request by jeandeaual**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28040)

### CODE REVIEW

**Review by quhxl — CHANGES_REQUESTED** [review]
The `Version` has to match the entry in the control panel.

image

File: manifests/p/PCSensor/FootSwitch/7.3.4/PCSensor.FootSwitch.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PCSensor.FootSwitch
+PackageVersion: 7.3.4
```
Comment (ItzLevvie) [inline manifests/p/PCSensor/FootSwitch/7.3.4/PCSensor.FootSwitch.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.123
```

File: manifests/p/PCSensor/FootSwitch/7.3.4/PCSensor.FootSwitch.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PCSensor.FootSwitch
+PackageVersion: 7.3.4
```
Comment (ItzLevvie) [inline manifests/p/PCSensor/FootSwitch/7.3.4/PCSensor.FootSwitch.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.123
```

File: manifests/p/PCSensor/FootSwitch/7.3.4/PCSensor.FootSwitch.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: PCSensor.FootSwitch
+PackageVersion: 7.3.4
```
Comment (ItzLevvie) [inline manifests/p/PCSensor/FootSwitch/7.3.4/PCSensor.FootSwitch.yaml:5]:
```suggestion
PackageVersion: 2.0.123
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**